pkg/apis/core/validation/validation_test.go

```diff
@@ -22726,15 +22726,23 @@ func TestValidateTopologySpreadConstraints(t *testing.T) {
 			MatchLabelKeys:    []string{"foo"},
 			LabelSelector: &metav1.LabelSelector{
 				MatchExpressions: []metav1.LabelSelectorRequirement{
+					// This one should be created from MatchLabelKeys.
 					{
 						Key:      "foo",
 						Operator: metav1.LabelSelectorOpNotIn,
-						Values:   []string{"value1", "value2"},
+						Values:   []string{"value1"},
+					},
+					{
+						Key:      "foo",
+						Operator: metav1.LabelSelectorOpNotIn,
+						Values:   []string{"value2", "value3"},
 					},
 				},
 			},
 		}},
-		wantFieldErrors: field.ErrorList{field.Invalid(fieldPathMatchLabelKeys.Index(0), "foo", "exists in both matchLabelKeys and labelSelector")},
+		// TODO: This expected result should be reverted after #129900 is merged.
+		// wantFieldErrors: field.ErrorList{field.Invalid(fieldPathMatchLabelKeys.Index(0), "foo", "exists in both matchLabelKeys and labelSelector")},
```

Comment:
Generally speaking, comment out the implementation isn't a great idea. You can just say:

```suggestion
		// TODO: This expected message is not perfect, and will be fixed in #129900.
```

Reply:
> Generally speaking, comment out the implementation isn't a great idea.

Thank you for your advice.
BTW I have one question about your suggestion.
Is  the comment "will be fixed in #129900" appropriate?
This fix is not included in #129900 because #129900 has not be implemented PodTopologySpread's matchLabelKeys.

Reply:
I don't quite get what you're trying to mean, but https://github.com/kubernetes/kubernetes/pull/129900 is gonna fix `validateMatchLabelKeysAndMismatchLabelKeys` which is used for Pod Topology Spread in this PR, right?
So, if this PR is merged before #129900, you ought to modify #129900 to delete this comment and fix the test case. If #129900 is merged before this PR, you can just remove this TODO and modify the test case.

Reply:
Oh, sorry, I was a bit confused...
Upon reconsidering, I understand it correctly now.